models/reviews.models.js

```diff
@@ -81,3 +81,37 @@ exports.insertComment = (review_id, username, body) => {
       return result.rows[0];
     });
 };
+
+exports.alterVotes = (review_id, inc_votes) => {
+  if (!inc_votes || !/^-?[0-9]*$/.test(inc_votes)) {
+    return Promise.reject({
+      status: 400,
+      msg: "Invalid patch object",
+    });
+  }
+
+  return checkExists("reviews", "review_id", review_id)
```

Comment:
There is no explicit need to check if the review exists - you would be able to check how many rows have been affected on the result object